addons/equipment/cfgUnitInsignia.hpp

```diff
@@ -0,0 +1,53 @@
+class CfgUnitInsignia
+{
+	class ga_insignia_de
+	{
+		displayName = "Germany"; // Name displayed in Arsenal
+		author = "Synch"; // Author displayed in Arsenal
+		texture = "x\grad\addons\equipment\data\insignia\de_v2.paa"; // Image path
```

Comment:
Yea, I'm getting the `x\grad\addons\...` error when opening the dynamic groups menu to test the insignia. It's probably only missing the leading `\` in the path, like this:

```
texture = "\x\grad\addons\equipment\data\insignia\de_v2.paa";
```

We have some cfgUnitInsignia stuff here:
https://github.com/gruppe-adler/gruppe_adler_mod/blob/master/addons/dynGroups/cfgUnitInsignia.hpp

We can either move all this stuff to `dynGroups`, or move the `dynGroups` stuff to `equipment`. I don't care, it fits both equally well imo.

Backpacks seem to work.

Reply:
see https://github.com/gruppe-adler/gruppe_adler_mod/pull/90/commits/522060449379fc19de3958bc49dabfa1eb6a3771